### ISSUE TITLE
Add a conda-forge as a test during pre-tag testing

### ISSUE THREAD

**Issue by esc** [numba/numba#9271]
During the next release (0.59.0) -- run the pre-tag test (testing the commit that would be tagged as a release) through conda-forge.

**Comment by mbargull** [comment]
(An aside regarding the question "Should some of the core Numba folks request access to CF feedstocks for LLVM/llvmlite/Numba so as to be able to help with/fix issues more easily?" from the linked meeting notes:
It's almost always much appreciated to have upstream devs involved in the package maintenance.
Even if it just means that they can be made aware of issues/changes by being notified of feedstock changes or team pings, that's already a win.
Please do feel encouraged to let us know if any of you would like to be added as maintainer to the `numba`/`llvmlite` feedstocks!)

**Comment by esc** [comment]
> (An aside regarding the question "Should some of the core Numba folks request access to CF feedstocks for LLVM/llvmlite/Numba so as to be able to help with/fix issues more easily?" from the linked meeting notes: It's almost always much appreciated to have upstream devs involved in the package maintenance. Even if it just means that they can be made aware of issues/changes by being notified of feedstock changes or team pings, that's already a win. Please do feel encouraged to let us know if any of you would like to be added as maintainer to the `numba`/`llvmlite` feedstocks!)

@mbargull yes, thank you! We will reach out when the majority of the issues in the NEXT milestone are complete and when we begin to prep things for the release.

**Comment by sklam** [comment]
@gmarkall did https://github.com/conda-forge/numba-feedstock/pull/131 for numba 0.59.0rc1

**Comment by esc** [comment]
Maybe for 0.62

**Comment by esc** [comment]
we decided to do this post RC1 for 0.62

**Comment by esc** [comment]
Postponed to post rc1 for 0.62

**Comment by swap357** [comment]
Will pre-tag test numba `0.65.0rc1` on conda-forge feedstock.

**Comment by esc** [comment]
Closing this. We settled on running any release candidate through conda-forge to catch any additional issues that arise on conda-forge only -- so no need to pre-tag test it there too.
